docs/user/access-control.md

```diff
@@ -53,24 +53,12 @@ kubectl create -n default rolebinding example-view \
 #### Write access to Applications within a namespace
 
 In order to create, update and delete Applications in a namespace, apply the
-`edit` ClusterRole in the desired namespace and the `kubeapps-repositories-read`
+`edit` ClusterRole in the desired namespace and the `$RELEASE_NAME-repositories-read`
 Role in the namespace Kubeapps is installed in. The `edit` ClusterRole should be
 available in most Kubernetes distributions, you can find more information about
 that role
 [here](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles).
 
-Additionally, we need to create a role to give read access to App Repositories and bind it to our service account.
-
-```
-export KUBEAPPS_NAMESPACE=kubeapps
-kubectl apply -n $KUBEAPPS_NAMESPACE -f https://raw.githubusercontent.com/kubeapps/kubeapps/master/docs/user/manifests/kubeapps-repositories-read.yaml
-kubectl create -n default rolebinding example-edit \
```

Comment:
sorry, even though it's not necessary to create the role (line above) it's necessary to create both rolebindings below

Reply:
Let's create rolebinding for the service-account in the chart - then the docs are correct, as what's left to do is to bind any users/groups according to the docs.

Reply:
The problem with the role binding is that it depends on the namespace and the service account name that the user is going to use, in this example is `default:example` but the user will need to modify that for other users.

Reply:
Not sure I follow.
The serviceaccount for kubeapps is created here: https://github.com/kubeapps/kubeapps/blob/master/chart/kubeapps/templates/apprepository-serviceaccount.yaml - and if bound to the role that one is OK.

What is left then is to bind selected users/groups to the role too, and this is a manual task, as documented.

Reply:
It's a bit complicated. The service account your are pointing to (`apprepository-serviceaccount`) is the one used by the `apprepository-controller`. That service account is properly bound to the [role](https://github.com/kubeapps/kubeapps/blob/master/chart/kubeapps/templates/apprepository-rbac.yaml) necessary to manage apprepositories. Note that this role have more privileges than the "read" ones.

What this documentation says is that the user account created in the [first step](https://github.com/kubeapps/kubeapps/blame/master/docs/user/access-control.md#L22) should be bound to the `apprepositories-read` and `apprepositories-write` roles that you have uploaded in this PR. If the user is not using the namespace `default` or the account `example` he should adapt the command and that's why we cannot pre-create the bindings for them.

Reply:
Yes -  I get that.
So to sum up.
Two roles are pre-created, it is the up to the user to bind accounts to to them with rolebindings, as described in the documentation:

```
In order to list the configured App Repositories in Kubeapps, [bind users/groups Subjects](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#command-line-utilities) to the `$RELEASE_NAME-repositories-read` role in the namespace Kubeapps was installed into by the helm chart.

#### Write access to App Repositories

In order to create and refresh App Repositories in Kubeapps, create and apply the
`kubeapps-repositories-write` Role in the namespace Kubeapps is installed in.
Likewise to the read access bind users/group Subjects to the
`$RELEASE_NAME-repositories-write` Role in the namespace Kubeapps is installed in
for users to create and refresh App Repositories in Kubeapps
```

Reply:
I think what @andresmgot is getting at here is we should add back the command instructions, rather than just have the verbal instructions, i.e.:

```
export KUBEAPPS_NAMESPACE=kubeapps
export KUBEAPPS_RELEASE_NAME=kubeapps
kubectl create -n default rolebinding example-edit \
  --clusterrole=edit \	
  --serviceaccount default:example	
kubectl create -n $KUBEAPPS_NAMESPACE rolebinding example-kubeapps-repositories-read \	
  --role=$KUBEAPPS_RELEASE_NAME-repositories-read \	
  --serviceaccount default:example
```

I've updated it to use the `$KUBEAPPS_RELEASE_NAME-repositories-read` role that is created by the chart instead of the one that was previously being created here by the `kubectl apply -f` command.

The other documentation should be updated similarly.